src/lib/uTop_compat.ml

```diff
@@ -67,7 +67,28 @@ let rec is_persistent_path = function
 #endif
 
 let invalid_package_error_to_string err =
-#if OCAML_VERSION >= (5, 2, 0)
+#if OCAML_VERSION >= (5,3,0)
```

Comment:
this section can be nested in the original one. Only a two lines are actually different

Reply:
I would suggest  to define `inline_code` compatibility helper rather than deduplicate the full printer.

Reply:
I thought only the newest cppo had nesting support, so I avoided doing it. Am I wrong?